test/xpu_api/numerics/complex.number/cmplx.over/pow.pass.cpp

```diff
@@ -40,7 +40,7 @@ test(T x, const dpl::complex<U>& y)
 {
     typedef decltype(promote(x) + promote(dpl::real(y))) V;
     static_assert((std::is_same<decltype(dpl::pow(x, y)), dpl::complex<V> >::value), "");
-    assert(dpl::pow(x, y) == dpl::pow(dpl::complex<V>(x, 0), dpl::complex<V>(y)));
+    is_about(dpl::pow(x, y), dpl::pow(dpl::complex<V>(x, 0), dpl::complex<V>(y)));
```

Comment:
We change assert(x == y) to is_about(x, y) (in this and in other places) because in release builds because in release build mode some calculations may give very close but not equivalent results